extensions/wikia/UserLogin/css/UserLogin.scss

```diff
@@ -75,3 +75,7 @@ $input-width: 230px;
 		display: none;
 	}
 }
+
+.hidden {
```

Comment:
I didn't say this rule gets overridden with `!important`, I said if we want to set a global `.hidden` rule, we'd want it to include `!important` so it would override more specific rules like `.my-module .my-div { display: none; }

I think we should create a separate ticket to do this work and ideally replace the per-extension instances of `.hidden`. We've done a fairly good job of sticking to the `.hidden` class name so it shouldn't be too hard to update.